<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMDDataBlockDialog</class>
 <widget class="QDialog" name="CMDDataBlockDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>描述</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="descriptionEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>名称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rateEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">1000</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>扫描周期(MS)</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="enableCheckBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>启用</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataTabNumberEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>数据表序号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>460</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>460</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshVarTabButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>99</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新数据表</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>471</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
